l10n_ch_dta/wizard/create_dta.py

```diff
@@ -1,10 +1,8 @@
 # -*- coding: utf-8 -*-
 ##############################################################################
 #
-#    Copyright Camptocamp SA
-#    Financial contributors: Hasa SA, Open Net SA,
-#                            Prisme Solutions Informatique SA, Quod SA
-#    Ported to v8.0 by Agile Business Group <http://www.agilebg.com>
+#    Copyright (c) 2015 brain-tec AG (http://www.braintec-group.com)
+#    All Right Reserved
```

Comment:
I am very sorry for that. I did not realized. I am fixing right now. Thanks for letting me know.